src/state/middlewares/system/system-status-notifier.js

```diff
@@ -5,17 +5,27 @@ import {
 } from 'ramda'
 
 import { notifySystemStatusChange } from '../../chatlist/actions'
-import { getAvailableLocales } from '../../operator/selectors'
+import {
+	getAvailableLocales,
+	hasOperatorRequestingChat,
+} from '../../operator/selectors'
 import { NOTIFY_SYSTEM_STATUS_CHANGE } from '../../action-types'
 
 const notifySystemStatus = ( { getState, dispatch } ) => next => action => {
 	if ( action.type === NOTIFY_SYSTEM_STATUS_CHANGE ) {
 		return next( action )
 	}
-	const previous = getAvailableLocales( getState() )
-	const result = next( action )
-	const current = getAvailableLocales( getState() )
-	if ( not( isEmpty( symmetricDifference( previous, current ) ) ) ) {
+
+	const previous = getAvailableLocales( getState() );
+	const previouslyRequestingChat = hasOperatorRequestingChat( getState() );
+	const result = next( action );
+	const current = getAvailableLocales( getState() );
+	const requestingChat = hasOperatorRequestingChat( getState() );
+
+	const localeAvailabilityChanged = ! isEmpty( symmetricDifference( previous, current ) );
+	const requestingChatChanged = requestingChat === previouslyRequestingChat;
```

Comment:
This should probably be `!==`, or the below action will be dispatched on every action